app/pathogen/arbovirus/analyze/columns.tsx

```diff
@@ -5,6 +5,7 @@ import React from "react";
 import { Button } from "@/components/ui/button";
 import { ArrowUpDown } from "lucide-react";
 import { pathogenColors } from "../dashboard/(map)/MapAndFilters";
+import validator from "validator";
```

Comment:
Commenting on random line for thread @HimanshuRanka 

> BUG: Go Arbovirus > Analysis > Assay - producer: HAI -> Causes the application to fail. Seems like other filters are working fine though.
> Only exists on analysis side.

I checked this one out a bit further and the url is actually `-` in airtable. My change appears to have affected the random records to be shown in the table at the start and the record with `-` just happened to be in the first 10. If you go to the real pathotracker and specify "assay: HAI" and go forward like six pages or something you actually get the same error which is interesting.

Anyways, resolved in f0ce3822feb2e878cb6876354cf4391e44f20034 so we won't have to think about this anymore :)

Reply:
Oh shit I had no idea about that LOL, guess that was a good catch for us then